Gopkg.toml

```diff
@@ -309,3 +309,7 @@ required = ["github.com/centrifuge/centrifuge-ethereum-contracts", "github.com/r
 [[constraint]]
   name = "github.com/urfave/cli"
   version = "1.20.0"
+
+[[constraint]]
+  branch = "master"
```

Comment:
Can we lock here a version or revision? In case that we need to regenerate the lock file.